src/common/snippets/src/pass/group_norm_tokenization.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/pass/group_norm_tokenization.hpp"
+
+#include "snippets/itt.hpp"
+#include "snippets/op/subgraph.hpp"
+#include "snippets/utils.hpp"
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+
+ov::snippets::pass::TokenizeGroupNormSnippets::TokenizeGroupNormSnippets() {
+    MATCHER_SCOPE(TokenizeGroupNormSnippets);
+
+    auto group_norm_pattern = ov::pass::pattern::wrap_type<ov::op::v12::GroupNormalization>();
+
+    ov::matcher_pass_callback callback = [=](ov::pass::pattern::Matcher& m) {
+        OV_ITT_SCOPED_TASK(ov::pass::itt::domains::SnippetsTransform, "Snippets::pass::TokenizeGroupNormSnippets")
+        auto group_norm_node = ov::as_type_ptr<ov::op::v12::GroupNormalization>(m.get_match_root());
+        if (group_norm_node->is_dynamic())
```

Comment:
Could you please add check for supported precision by [Tokenization](https://github.com/openvinotoolkit/openvino/blob/5e7e51dc77822ab6cb9e45c7208100c57bec4104/src/common/snippets/src/pass/collapse_subgraph.cpp#L221) as well? Because all specific tokenization passes should collapse nodes only if they have supported element types.

Reply:
Precision limitation f32 is added, will extend after fix input precision convert error!

Reply:
supported precision by [Tokenization](https://github.com/openvinotoolkit/openvino/blob/5e7e51dc77822ab6cb9e45c7208100c57bec4104/src/common/snippets/src/pass/collapse_subgraph.cpp#L221) is added. Thank you Alexandra!